src/components/dataset-summary/row.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="dataset-row">
+<div class="row">
+            <div class="col-xs-6">
+                <span class="dataset-name">{{ dataset.name }}</span>
+                <span v-if="dataset.isNew" class="dataset-new">
+                    <span class="icon-plus-circle"></span>
+                    new!
+                </span>
+</div>
+            <div class="col-xs-6 properties-count">
+                {{ inFormProperties.length }} of {{ dataset.properties.length }} properties
+                <a href="javascript:void(0)" role="button" class="expand-button" @click.prevent="toggleExpanded">
+                    <span v-if="!expanded" class="icon-chevron-right"></span>
+                    <span v-else class="icon-chevron-down"></span>
+                </a>
+            </div>
+        </div>
+        <div v-show="expanded" class="row property-list">
+            <div class="col-xs-12">
+                <span v-for="(property,index) in inFormProperties" :key="property.name">
+                    <span>{{ property.name }}</span>
+                    <span v-if="property.isNew" class="icon-plus-circle property-new"></span>
+                    <span v-if="index < inFormProperties.length - 1">, </span>
```

Comment:
I like the idea of moving this into Transifex so that translators can make this decision without us having to research it!

That said, it's not just the comma that's the issue, it's also the space. Japanese is typically written without space characters, including after punctuation. You can see that the Japanese punctuation characters "、" and "。" have built-in white space (the punctuation looks left-aligned). Here's a list of Japanese city names that doesn't contain any space characters: 東京、大阪、札幌

Ideally, translators would add a space after the comma in `common.punctuations.comma` if their language uses that. However, I think it'd be pretty hard to communicate that ask to translators. I'm also not sure that it'd work with our current Transifex workflow, which does some automatic trimming of white space.

I propose that we use `common.punctuations.comma` like you're doing, then reuse our `<sentence-separator>` component for the possible space after the comma:

```vue
<span v-if="index < inFormProperties.length - 1">{{ $t('common.punctuations.comma' ) }}<sentence-separator/></span>
```

If we go that approach, we could think about renaming `<sentence-separator>` to something more generic at some point, maybe `<i18n-space>`.

I also want to mention though that we can totally come back to this later. If you want to run with `, `&nbsp;for now, that'd work fine for this release.

Reply:
@matthew-white given the depth you're getting into this, you may not want to conflate the space-after-comma vs the space-after-full-stop.  I wouldn't be surprised if there were expectations in some languages/locales that the space after a comma-equivalent ("list separator"?) is a different size to the space after a full-stop.  That was the case historically in English.